docs/source/Plugin/P092_UVR61-3v8_3.rst

```diff
@@ -0,0 +1,85 @@
+.. include:: ../Plugin/_plugin_substitutions_p09x.repl
+.. _P092_UVR61-3v8_3_page:
+
+UVR61-3 (v8.3 or higher)
+========================
+
+|P092_typename|
+|P092_status|
+
+Introduction
+------------
+
+The UVR61-3 unit is a three-circuit universal control system.
+
+The UVR61-3 unit is designed for an energy-saving and cost-effective drying of cellars and other parts of buildings by means of fan control.
+
+The UVR61-3 three-circuit controllers has various thermostat, differential temperature and speed control functions for use in solar and heating systems, too.
+
+* This data frame for UVR61-3 has been changed starting from firmware v8.3 or higher.
+* The DL bus runs at 12 volts.
+* The data frame consist of 61 bytes (excluding sync and CRC).
+* The data are transmitted @ 488Hz, therefore the transmission of one data set takes about 1.3 seconds.
+
+.. image:: P092_UVR61-3.png
```

Comment:
Are these images (more below) already committed earlier? As I can't find them all in this PR...

Reply:
Yes, they are already in the latest release.